### ISSUE TITLE
Can't resolve 'brazilian-values' in 'amazing-react-charts\dist\core'

### ISSUE THREAD

**Issue by davwheat** [nginformatica/amazing-react-charts#7]
Hi! I'm getting this error while trying to use this library. I don't plan to use any Brazilian values, so is there a way to use this package without installing `brazilian-values`? Many of the charts import this at the moment.

```
ERROR #98124  WEBPACK

Generating development JavaScript bundle failed

Can't resolve 'brazilian-values' in 'C:\...\node_modules\amazing-react-charts\dist\core'    

If you're trying to use a package make sure that 'brazilian-values' is installed. If you're trying to use a local file  
make sure that the path is correct.

File: node_modules\amazing-react-charts\dist\core\StackedBarChart.js
```

**Comment by leorigon** [comment]
Hi, @davwheat. 

We can change it, allowing a prop that's receive a function to change the monetary values. It's necessary to remove this package from the library to not occur the error again.

With you interested in change it, we'll apreciate it. If not, on the next weeks, I can solve it.

**Comment by leorigon** [comment]
This issue was fixed on PR #14, brazilian-values dependency is not needed to use the library.
